R/baggr.R

```diff
@@ -224,6 +224,13 @@ baggr <- function(data,
   # if(!inherits(data, "data.frame") || nrow(data) == 1)
     # stop("data argument must be a data.frame of >1 rows")
 
+  # get label agrument and pass it as effect for compatibility
+  if("effect" %in% attributes(list(...))$names){
```

Comment:
You need to write a few automated unit tests to prove that passing `effect` argument works the same way as `label`.

Reply:
Got it! There a few existing tests that used the `effect` argument, so I added tests to check that `label` and `effect` can be passed interchangeably there.